src/display/editor/tools.js

```diff
@@ -741,6 +744,30 @@ class AnnotationEditorUIManager {
     this.#container.focus();
   }
 
+  findParent(x, y) {
+    for (const layer of this.#allLayers.values()) {
+      const {
+        x: layerX,
+        y: layerY,
+        width,
+        height,
+      } = layer.div.getBoundingClientRect();
+      if (
+        x >= layerX &&
+        x <= layerX + width &&
+        y >= layerY &&
+        y <= layerY + height
+      ) {
+        return layer;
+      }
+    }
+    return null;
+  }
+
+  enableUserSelect(value) {
+    this.#viewer.classList.toggle("noUserSelect", !value);
+  }
```

Comment:
Can we please "invert" this function instead, since that seems less risky with future changes, such that if you'd ever forget to add the parameter at a call-site we'd fallback to always *enable* user-select?

```suggestion
  disableUserSelect(value = false) {
    this.#viewer.classList.toggle("noUserSelect", value);
  }
```